SEP-0012.md

```diff
@@ -288,7 +296,16 @@ class NDCubeABC(astropy.nddata.NDDataBase):
              wcs: Optional[Union[BaseHighLevelWCS, ExtraCoordsABC]] = None
              ) -> NDCubeABC:
         """
-        Crops an NDCube given real world coords of lower and upper corners of a region of interest.
+        Crop given world coordinate objects describing the lower and upper corners of a region.
+
+        The region of interest is defined in pixel space, by converting the world
+        coordinates of the corners to pixel coordinates and then cropping the
+        smallest pixel region which contains the corners specified.
+        This means that the edges of the world coordinate region specified by
+        the coordinates are not gaureented to be included in the cropped output.
+        This is normally noiticable when cropping a celestial coordinate in a
+        frame which differs from the native frame of the coordinates in the WCS.
```

Comment:
What impact does defining this in the SEP have on future changes to ndcube which more rigorously identifies the region of interest, e.g. one that does include the whole region defined in a different coordinate frame?

Reply:
> not gaureented to be included
I don't see an issue if in future they are then that ok? But removal is fine too, I would probably fix the spelling if it is staying guaranteed 😆

Reply:
How about this?
```
The region of interest is defined in pixel-space based on the world coordinates input as the region's corners.
```

Reply:
I think I am pro that, want to add a suggestion for that?

(To be clear I have added a version of this sentence to NDCube proper)

Reply:
I have just pushed a commit with this wording in, and an extra sentence being explicit about the fact we aren't specifying the algorithm in here.